src/governance.rst

```diff
@@ -0,0 +1,172 @@
+reinstate the activity.
+
+Sub-teams have a charter that is either *dynamic* or *static*.
+
+* A *dynamic* charter means that the sub-team is self-organizing, with respect
+  to its own internal policies, procedures, and membership. A sub-team may choose
+  to modify its membership independent of the core team. For example, a
+  Google Summer of Code team could be a good candidate for a dynamic charter.
+  Alternatively, language-based maintenance teams also have a dynamic charter.
+* A *static* charter means that all membership decisions and non-trivial policies
+  changes must be approved by the core team. For example, a finance team
+  may require a static charter.
+
+All sub-teams must adhere to the governance, policies, and procedures of
+conda-forge at all times.
+
+Voting
+------
+This section presents descriptions and criteria for voting items in the
+conda-forge community. The core team is the only team with voting rights.
+Members of core may also call a vote on any topic. The restrictions on
+calling a vote are as follows:
+
+* There must only be one vote active on a particular item at any time.
+* The act of calling for a vote cannot itself violate the code of
+  conduct. For example, Sam repeatedly called for votes immediately
+  after a previous votes failed to achieve Sam's result. Sam is
+  attempting to bully other members of core into agreeing, and is thus
+  violating the code of conduct.
+
+Voting items are labeled as either **standard** or **sensitive**.
+Standard items are ones where public record and discourse and record is
+preferable. Sensitive voting items are ones where the results of the
+vote should remain private to the voters after the vote has occurred.
+Sensitive votes should take place on
+`the Helios voting system <https://vote.heliosvoting.org/>`_ in order
+retain anonymity.
+
+The default voting period is 1 week (7 days). This may be modified at
+the time when a vote is called, but may never be less than 24 hrs.
+
+:Nominate new member of staged-recipes: The proposer must provide
+    a brief justification as to why the new member is desirable or needed.
+
+    * Sensitive
+    * 50% Majority to pass
+
+:Nominate new member of core: The proposer must provide
+    a sufficient justification as to why the nominee should be welcomed
+    into core. It is necessary that all nominees are current and active
+    members of staged-recipes. Nominees to core must have been members
+    of staged-recipes for at least 1 month (4 weeks) prior to this vote.
+
+    * Sensitive
+    * 66.7% Majority to pass
+
+:Sub-team Formation: Proposers must specify the name, role & responsibility,
+    members, and charter (dynamic or static) of any new sub-teams.
+
+    * Standard
+    * 50% Majority to pass
+
+:Sub-team Dissolution: Proposers must specify the name and justification
+    for why a sub-team should be dissolved.
+
+    * Standard
+    * 50% Majority to pass
```

Comment:
If there are no `core` members on the team, doesn't it dissolve anyways? How do we handle this situation?